docs/linelist.rst

```diff
@@ -0,0 +1,123 @@
+.. highlight:: rest
+
+.. _LineList:
+
+******************
+LineList Class
+******************
+
+.. index:: LineList
+
+Overview
+========
+
+This Class is designed to organize and handle information about atomic
+and/or molecular transition lines (e.g. HI Lya, CIV 1548, Hydrogen
+Balmer series, etc.) observed in a variety of astrophysical
+environments. It is currently implemented for absorption lines, but we
+expect to also include common emission lines in the near future.
+
+..
+   (:ref:`AbsLine Class`).  add this back in when written
+
+By definition, a LineList is a unique collection of transitions lines
+specified by:
+
+
+============ ========= =========== ====================================================
+Property     Variable  Type        Description
+============ ========= =========== ====================================================
+LineList key llst_keys str or list A key to define a subsample of transitions to load:
+                                     'ISM' - "All" ISM lines (can be overwhelming!)
+                                     'Strong' - Strong ISM lines
+                                     'HI' - HI Lyman series
+                                     'H2' - H2 (Lyman-Werner)
+                                     'CO' - CO UV band-heads
+                                     'EUV' - Extreme UV lines
+============ ========= =========== =====================================================
```

Comment:
I think the table and the sentence above it is confusing. How about a list of bullet points, for 'ISM', 'strong', etc?